packages/message-kit/src/lib/skills.ts

```
@@ -0,0 +1,261 @@
+          valueFound = true;
+        }
+      } else if (type === "number") {
+        // Handle comma-separated numbers
+        const numberParts = parts.reduce<number[]>((acc, part, idx) => {
+          if (!usedIndices.has(idx) && !isNaN(parseFloat(part))) {
+            usedIndices.add(idx);
+            const numbers = part
+              .split(",")
+              .map((n) => parseFloat(n.trim()))
+              .filter((n) => !isNaN(n));
+            acc.push(...numbers);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)